extensions/control-plane/store/sql/contract-negotiation-store-sql/docs/schema.sql

```diff
@@ -45,7 +45,7 @@ CREATE TABLE IF NOT EXISTS edc_contract_negotiation
     correlation_id       VARCHAR,
     counterparty_id      VARCHAR                                            NOT NULL,
     counterparty_address VARCHAR                                            NOT NULL,
-    protocol             VARCHAR DEFAULT 'ids-multipart'::CHARACTER VARYING NOT NULL,
+    protocol             VARCHAR                                            NOT NULL,
```

Comment:
no default anymore?

Reply:
Given that the protocol is a mandatory field I see no point on having a default value